{"event": "verbose", "uuid": "6fbb55b9-3087-4c00-9ada-3c9365fd74e1", "counter": 3, "stdout": "\u001b[0;31mThe error appears to be in '/tmp/tmpuha05et8.yml': line 213, column 7, but may\u001b[0m", "start_line": 2, "end_line": 3, "runner_ident": "a4237ed1-fb2e-42d6-a5bc-6a0b762e4da2", "created": "2025-05-15T18:09:43.956350+00:00"}